### ISSUE TITLE
Accessibility Issue - Dropdown values are not visible properly

### ISSUE THREAD

**Issue by Anagha20130362** [topcoder-platform/Training-Challenges#335]
Bug Title: Accessibility Issue - Dropdown values are not visible properly

Steps to Reproduce: 

1. Navigate to https://www.topcoder.com/#do-box-block_6237f8de0dd83
2. Click on 'Let's Talk' link under PROJECTS & PROFESSIONAL SERVICES section
3. Click on the first dropdown 'What are you interested in?'
4. Scroll the page and click on the dropdown.

Actual Result: The dropdown values are sometimes displayed below the page , above the page or middle of the page. When user scrolls the page the values are displayed in different places on the page.

Expected Result: The dropdown values should be display in a constant place rather than changing its position.

Screenshots or screencast URL: <

![image](https://user-images.githubusercontent.com/102279071/161490371-ba92aaa2-6391-4c34-b2d2-acb2a286c41f.png)

![image](https://user-images.githubusercontent.com/102279071/161490478-09af070e-9937-41ea-bc04-90e70ca6af9b.png)

![image](https://user-images.githubusercontent.com/102279071/161490531-d1afea37-1267-4945-92da-ace93039b92e.png)

Device/OS/Browser Information:

Device:  L-356016884
OS: Windows 10 Enterprise version 1909, OS Build: 18363.1801
Browser:  Google Chrome version 99.0.4844.82

**Comment by sdgun** [comment]
Valid issue. But this is not an accessibility issue, it's a functional issue.
